### PR TITLE
i18n(es): translate `badges`

### PR DESCRIPTION
**Pull request by dreyfus92**
#### Description

- What does this PR change? Give us a brief description.
Translates `badges` to spanish

### CODE REVIEW

File: docs/src/content/docs/es/components/badges.mdx [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+Para usar un color de insignia integrado, establece el atributo [`variant`](#variant) en uno de los valores admitidos.
+
+<Preview>
+
+```mdx
+import { Badge } from '@astrojs/starlight/components';
+
+<Badge text="Nota" variant="note" />
+<Badge text="Éxito" variant="success" />
+<Badge text="Tip" variant="tip" />
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/badges.mdx:38]:
```suggestion
<Badge text="Consejo" variant="tip" />
```

Was this missed or expected? Took the translation from https://github.com/withastro/starlight/blob/main/packages/starlight/translations/es.json#L24C17-L24C24

File: docs/src/content/docs/es/components/badges.mdx [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+<Badge text="Precaución" variant="caution" />
+<Badge text="Peligro" variant="danger" />
+```
+
+<Fragment slot="markdoc">
+
+```markdoc
+{% badge text="Nota" variant="note" /%}
+{% badge text="Éxito" variant="success" /%}
+{% badge text="Tip" variant="tip" /%}
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/badges.mdx:48]:
```suggestion
{% badge text="Consejo" variant="tip" /%}
```

File: docs/src/content/docs/es/components/badges.mdx [hunk 3]
```diff
@@ -0,0 +1,150 @@
...
+{% badge text="Precaución" variant="caution" /%}
+{% badge text="Peligro" variant="danger" /%}
+```
+
+</Fragment>
+
+<Fragment slot="preview">
+	<Badge text="Nota" variant="note" />
+	<Badge text="Éxito" variant="success" />
+	<Badge text="Tip" variant="tip" />
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/badges.mdx:58]:
```suggestion
	<Badge text="Consejo" variant="tip" />
```

File: docs/src/content/docs/es/components/badges.mdx [hunk 4]
```diff
@@ -0,0 +1,150 @@
...
+<Badge
+	slot="preview"
+	text="Personalizado"
+	variant="success"
+	style={{ fontStyle: 'italic' }}
+/>
+
+</Preview>
+
+## Props del componente `<Badge>`
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/badges.mdx:126]: In https://github.com/withastro/starlight/pull/2395, this section was translated as "## Props de `<Aside>`". I guess we should figure out the best one and always use the same syntax for consistency.

Do you think this one is better and the other should be updated or the other way around?

File: docs/src/content/docs/es/components/badges.mdx [hunk 5]
```diff
@@ -0,0 +1,150 @@
...
+
+**Implementación:** [`Badge.astro`](https://github.com/withastro/starlight/blob/main/packages/starlight/user-components/Badge.astro)
+
+El componente `<Badge>` acepta las siguientes propiedades y también cualquier [otro atributo `<span>`](https://developer.mozilla.org/es/docs/Web/HTML/Atributos_globales):
+
+### `text`
+
+**requerido**
```
Comment (HiDeoo) [inline docs/src/content/docs/es/components/badges.mdx:134]:
```suggestion
**requerido**  
```

Missing line break

**Review by HiDeoo — CHANGES_REQUESTED** [review]
Thanks for the translation :raised_hands:

As a non-native speaker, I reviewed this PR using the [Docs Translation Review Checklist](https://hackmd.io/1KYnrpUQR0SAGTzHIUoUDg?view). I left a few questions and comments but otherwise, the translation looks good to me.

File: docs/src/content/docs/es/components/badges.mdx (hunk 4 above)
Comment (dreyfus92) [reply docs/src/content/docs/es/components/badges.mdx:126]: Both could work

**Comment by dreyfus92** [comment]
Thanks for the review @HiDeoo

**Review by HiDeoo — APPROVED** [review]
Thanks for updating the PR :rocket:

**Pull request merged**